evennia/utils/evmenu.py

```diff
@@ -1052,12 +1052,14 @@ def node_formatter(self, nodetext, optionstext):
         if self._session:
             screen_width = self._session.protocol_flags.get(
                     "SCREENWIDTH", {0: _MAX_TEXT_WIDTH})[0]
+            logger.log_msg("Flags: %s" % str(self._session.protocol_flags))
```

Comment:
These look like a leftover debug message; no need to have this in the final code?